File: modul 4/PRAK405_2410817120001_Putri_Fatima_Azhara.c
#include <stdio.h>

int main() {
    int x, y;
    scanf("%d %d", &x, &y);

    for (int i = 1; i <= x; i++) {
        int hasil = 0;
        for (int z = i; z > 0; z--) {
            if (z > 1) {
                printf("(%d * %d) + ", z, y);
            } else {
                printf("(%d * %d)", z, y);
            }
            hasil += z * y;
        }
        printf(" = %d\n", hasil);
    }

    int A = 1, B = 1, C = 0;
    while (A <= x) {
        C += B * y;
        B += A + 1;
        A++;
    }
    printf("%d\n", C);

    return 0;
}

File: modul 4/PRAK405_2410817120001_Putri Fatima Azhara.py
x, y = map(int, input().split())

for i in range(1, x + 1):
    hasil = 0
    while i > 1:
        print(f"({i}*{y}) + ", end="")
        hasil += i * y
        i -= 1
    print(f"({i}*{y})", end="")
    hasil += i * y
    print(f" = {hasil}")

A = 1
B = 1
C = 0
while A <= x:
    C += B * y
    B += A + 1
    A += 1
print(C)